Library/ViewModels/CommentsViewModel.swift

```diff
@@ -196,12 +196,29 @@ public final class CommentsViewModel: CommentsViewModelType,
       // Thread hop so that we don't circularly buffer.
       .ksr_debounce(.nanoseconds(0), on: AppEnvironment.current.scheduler)
 
+    // Allow empty arrays from the first emission.
+    let emptyCommentsWithInitialProject = Signal.zip(comments, initialProject)
+      .filter { $0.0.isEmpty }
```

Comment:
I think we should use the variable names instead of $0.0 and $0.1 for readability in line 201 and 202.